doc/clone-datavolume.md

```diff
@@ -21,12 +21,7 @@ spec:
     pvc:
       namespace: source-ns
       name: source-datavolume
-  pvc:
-    accessModes:
-      - ReadWriteOnce
-    resources:
-      requests:
-        storage: 500Mi
+  storage:
```

Comment:
does this need a `{}` in order to be valid yaml?

Reply:
Good catch :)